services/nginx-certbot/entrypoint.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
-export PRIMARY_DOMAIN=$(echo $DOMAIN_NAME | cut -d' ' -f1)
+# used by, e.g., nginx to find the certs, since the first name in
+# the list of domains is used to name the cert folder
+export PRIMARY_DOMAIN=$(echo $DOMAIN_NAMES | cut -d' ' -f1)
```

Comment:
So, the NGINX image on which this one is based performs environment variable substitution into config files that end in `.template` before starting anything else. We kind of need to rely on the templating feature since we specify the hostnames as env vars, and NGINX has no way of retrieving their values at runtime.

I had thought that this `export PRIMARY_DOMAIN` line would run before template substitution, but I was encountering it not getting populated and resorted to hardcoding it into the top-level `.env` file in the interests of time. I plan to go back and fix this, though, and remove `PRIMARY_DOMAIN` from the `.env` file, since it should always be the first element in `DOMAIN_NAMES`.